aws/kubectl-manifest/README.md

```diff
@@ -0,0 +1,42 @@
+# Kubeclt Manifest
+
+Cloud agnostic tf module w/ `kubectl` provider to apply arbitrary k8s manifests.
+
+## Inputs/Variables
+
+| Variable     | Description                                               | Example                                                        |
+| ------------ | --------------------------------------------------------- | -------------------------------------------------------------- |
+| install_name | The install id                                            | `{{.nuon.install.id}}`                                         |
+| cluster_name | The name of the cluster - sourced from the sandbox.       | `{{.nuon.install.sandbox.outputs.cluster.name}}`               |
+| iam_role_arn | The name of the role to invoke the `aws eks` cli cmd with | `{{.nuon.install.sandbox.outputs.runner.runner_iam_role_arn}}` |
```

Comment:
This is the biggest _practical_ difference between this being managed internally, vs here. The permissions can not be as locked down.